### PR TITLE
Rephrased nonzero to stored for sparse matrix display #19119

### PR DESCRIPTION
**Pull request by akaysh**
Is this all I need to do? I tested and ran it locally.

### CODE REVIEW

**Comment by akaysh** [comment]
@StefanKarpinski Please review this. Ty

File: base/sparse/sparsematrix.jl [hunk 1]
```diff
@@ -1406,7 +1406,7 @@ function broadcast{Tf}(f::Tf, A::SparseMatrixCSC)
     fpreszero = fofzero == zero(fofzero)
     return fpreszero ? _broadcast_zeropres(f, A) : _broadcast_notzeropres(f, fofzero, A)
 end
-"Returns a `SparseMatrixCSC` storing only the nonzero entries of `broadcast(f, Matrix(A))`."
+"Returns a `SparseMatrixCSC` storing only the stored entries of `broadcast(f, Matrix(A))`."
```
Comment (Sacha0) [inline base/sparse/sparsematrix.jl]: `nonzero` is correct here.

**Comment by Sacha0** [comment]
Welcome `@akaysh`! Modifying docstrings usually requires that you rebuild the documentation. Please see [CONTRIBUTING.md's "Improving documentation" section](https://github.com/JuliaLang/julia/blob/master/CONTRIBUTING.md#improving-documentation) for instructions. Thanks and best!

**Comment by fredrikekre** [comment]
Not a native speaker, but does it read more naturally as `3×3 sparse matrix with 3 stored Float64 entries` ?

**Comment by akaysh** [comment]
@Sacha0 I have updated the PR. Thanks for your help.

**Comment by Sacha0** [comment]
This looks great!

Minor additional suggestion: Prefer descriptive commit messages. For example, a good commit message after squashing could read "Change the `show` header for `SparseMatrixCSC`s from mentioning "nonzero entries" to "stored entries", and update impacted docstrings."

Best!

**Comment by akaysh** [comment]
@Sacha0 Okay sure. I will take care of it now. Ty
Why is the build job failing on OSX server though?

**Comment by Sacha0** [comment]
> Why is the build job failing on OSX server though?

Travis macOS has been unhappy lately. Don't worry about it :). Best!

**Comment by nalimilan** [comment]
Should be fixed now. Restarting the build.

**Comment by Sacha0** [comment]
Rebase? Thanks!

**Comment by akaysh** [comment]
@Sacha0 How should I resolve arrays.rst (delete/modify) conflict?

**Comment by tkelman** [comment]
the new doc system has markdown files instead of rst, and the docstrings are no longer duplicated. try just reverting your second commit.

**Comment by tkelman** [comment]
ah there's also a conflict in the .jl file but that should be simpler to resolve

**Comment by akaysh** [comment]
Okay! Done.

File: base/sparse/sparsematrix.jl [hunk 2]
```diff
@@ -33,10 +33,11 @@ Returns the number of stored (filled) elements in a sparse array.
 
 ```jldoctest
 julia> A = speye(3)
-3×3 sparse matrix with 3 Float64 nonzero entries:
-  [1, 1]  =  1.0
-  [2, 2]  =  1.0
-  [3, 3]  =  1.0
+3×3 sparse matrix with 3 Float64 stored entries:
+	[1, 1]  =  1.0
```
Comment (tkelman) [inline base/sparse/sparsematrix.jl]: the tabs were removed and should stay that way for this doctest to paas

**Comment by akaysh** [comment]
Okay that was not intended.  Updated the PR
Thanks.

File: base/sparse/sparsematrix.jl [hunk 3]
```diff
@@ -1466,7 +1466,7 @@ _maxnnzfrom(Cm, Cn, A) = nnz(A) * div(Cm, A.m) * div(Cn, A.n)
 @inline _checked_maxnnzbcres(Cm, Cn, As...) = Cm != 0 && Cn != 0 ? _unchecked_maxnnzbcres(Cm, Cn, As) : 0
 
 # _map_zeropres!/_map_notzeropres! specialized for a single sparse matrix
-"Stores only the nonzero entries of `map(f, Matrix(A))` in `C`."
+"Stores only the stored entries of `map(f, Matrix(A))` in `C`."
```
Comment (Sacha0) [inline base/sparse/sparsematrix.jl]: `nonzero` is correct here; please restore `nonzero`.

**Review by Sacha0 — APPROVED** [review]
@KristofferC @fredrikekre, would you be happy calling this pull request a reasonable approximation to optimal and merging as is? Thanks!

**Comment by fredrikekre** [comment]
The reason behind my comment was that I now read this as the entries would be "`Float64-stored`", whatever that means. And also, for longer `eltype`s the number `nnz` and "stored" are separated, feels more natural if "stored" is right after the number. But I leave it up to better judging people to decide what sounds best!

```jl
3×3 sparse matrix with 3 Complex{Float64} stored entries:
```

vs 

```jl
3×3 sparse matrix with 3 stored Complex{Float64} entries:
```

or perhaps even with commas:

```jl
3×3 sparse matrix with 3 stored, Complex{Float64}, entries:
```

**Comment by KristofferC** [comment]
I don't feel strongly about it. What you are likely looking for when you are reading the header is the number of nonzeros and what type they have. It therefore makes sense to have these close to each other (like the current PR does).  On the other hand, like @fredrikekre, it feels to me like what is stored are "Float64-entries".

Conclusion is that this is fine from my point of view :)

**Review by KristofferC — APPROVED** [review]

**Comment by Sacha0** [comment]
Cheers, perhaps @akaysh should make the call then? @akaysh, do you prefer the proposed form over the existing form, and if so would you enjoy implementing those changes? If to the contrary you prefer the existing form or are not excited about implementing those changes, let's merge as is. Thanks all!

**Comment by akaysh** [comment]
I feel what @KristofferC is saying seems more right. Lets merge this.
Thanks :) @Sacha0

**Pull request merged**